cmd/ddev/cmd/dotddev_assets/commands/host/launch

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 
 ## Description: Launch a browser with the current site
-## Usage: launch
-## Example: "ddev launch"
+## Usage: launch [path]
+## Example: "ddev launch" or "ddev launch mypath"
 
 case $OSTYPE in
   linux-gnu)
-    xdg-open ${DDEV_PRIMARY_URL}
+    xdg-open ${DDEV_PRIMARY_URL}/$1
```

Comment:
I think each of these would be better and simpler if we construct a single env var (FULLURL?) that includes the $1. Something like this at the top:
```
FULLURL=${DDEV_PRIMARY_URL}
if [ ! -z ${1:-} ] ; then FULLURL="${DDEV_PRIMARY_URL}/${1}"; fi
```
Then use FULLURL in these places. This avoids extra "/" in url in some cases as well

Reply:
Change is made, but the variant with the leading slash does not run here. The URL `https://typo3-8-mysql-8-test.ddev.site/C:/Program` is opened. I guess a kind of excaping issue but I don't know how to solve and if it's more a problem of the call or in the script

Reply:
Using two slahes works fine in git-bash on Windows